src/main/java/org/myrobotlab/service/abstracts/AbstractPinEncoder.java

```diff
@@ -78,37 +78,32 @@ public void setZeroPoint() {
     // pass the set zero point command to the controller
     controller.setZeroPoint(this);
   }
-
-  @Override
-  public void setController(EncoderController controller) {
-    this.controller = controller;
-  }
-
-  @Override
+  
   public void setPin(String pin) {
     this.pin = pin;
   }
 
-  @Override
   public void setPin(Integer address) {
     this.pin = String.format("%d", address);
   }
 
   @Override
   public void disable() {
-    // TODO Auto-generated method stub
-    
+    enabled = false;

```

Comment:
This isn't correct.. This should actually send a message to the Amt203 or As5048 encoders in MrlComm land to make sure the devices are disabled.  Otherwise data will continue to flow over the serial port from the arduino and could clog up communication.    this should be should be abstract.

Reply:
same as above